<launch>
    <let name="urdf_path"
         value="$(find-pkg-share lab4)/urdf/robot.xacro"/>
    <let name="rviz_path"
         value="$(find-pkg-share lab4)/rviz/conf.rviz"/>
         
    <node pkg="robot_state_publisher" exec="robot_state_publisher">
        <param name="robot_description"
                value="$(command 'xacro $(var urdf_path)')"/>
    </node>
    
    <node pkg="rviz2" exec="rviz2" output="screen"
    args="-d $(var rviz_path)"/>

    <node pkg="lab4" exec="inverse_kin"/>
</launch>